ubuntudesign/www.ubuntu.com
# update Ubuntu Advantage service description

## Done

* updated /legal/ubuntu-advantage/service-description#response-times with October changes
* whitespace clean-up

## QA

1. go to http://www.ubuntu.com-1607-service-description-update.demo.haus/legal/ubuntu-advantage/service-description
2. see that the text matches the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#heading=h.gf32urfikio3)
3. resolve comments in copy doc left in for the reviewer of this pull request

## Issue / Card

[trello card](https://trello.com/c/I2liAgWv)
